File: Backend/Infrastructure.MyShop/Models/DTO/ProductDTO/ProductEditDTO.cs
using Microsoft.AspNetCore.Http;

namespace Infrastructure.MyShop.Models.DTO.ProductDTO
{
    public class ProductEditDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int Price { get; set; }
        public List<IFormFile>? ImagesUpload { get; set; }
        public int? CategoryId { get; set; }

    }
}

File: Backend-Java/src/main/java/org/example/Infrastructure/dto/productDTO/ProductEditDTO.java
package org.example.Infrastructure.dto.productDTO;

import lombok.Data;
import org.springframework.web.multipart.MultipartFile;

import java.util.List;

@Data
public class ProductEditDTO {
    private int id;
    private String name;
    private String description;
    private double price;
    private List<MultipartFile> imagesUpload;
    private int categoryId;
}
